srcpkgs/ckan/template

```diff
@@ -0,0 +1,32 @@
+# Template file for 'ckan'
+pkgname=ckan
+version=1.27.0
+revision=1
+archs="x86_64"
+hostmakedepends="binutils bsdtar"
+makedepends="binutils bsdtar"
+depends="mono curl libcurl libcurl-32bit libgdiplus"
+short_desc="Client for CKAN - The Comprehensive Kerbal Archive Network"
+maintainer="lowranker <lowranker@protonmail.com>"
+license="MIT"
```

Comment:
Any reason why you don't attempt to build it from source?

Reply:
Couldn't build it from source on my machine, not quite ready to write a template without building manually first... Also going for unpacking .debs for simplicity